File: trail1/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(a, b) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int val1 = a < b ? 1 : 0;
        int val2 = a == b ? 1 : 0;
        Console.WriteLine(val1 + " " + val2);
    }
}

File: trail1/특정 조건 두 정수 비교/specific-comparison-of-two-natural-numbers.c
#include <stdio.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    int val1 = (a<b) ? 1 : 0;
    int val2 = (a==b) ? 1 : 0;
    printf("%d %d", val1, val2);
    return 0;
}